appservice/src/main/java/nu/yona/server/subscriptions/rest/UserController.java

```diff
@@ -372,6 +378,7 @@ public UserResource toResource(UserDTO user)
 					addWeekActivityOverviewsLink(userResource);
 					addNewDeviceRequestLink(userResource);
 					addAppActivityLink(userResource);
+					pinResetRequestController.addLinks(userResource);
```

Comment:
OK, indeed this seems necessary. But there is maybe a way to avoid having a `null` PinResetRequestController` in the constructor? Maybe an overload for the include private data case and an overload for the public case?